web/src/alert_words.ts

```diff
@@ -68,15 +92,23 @@ export function process_message(message) {
         );
     }
 }
+interface Messages {
+    sender_email: string;
+    alerted: boolean;
+}
 
-export function notifies(message) {
+export function notifies(message: Messages): boolean {
     // We exclude ourselves from notifications when we type one of our own
     // alert words into a message, just because that can be annoying for
     // certain types of workflows where everybody on your team, including
     // yourself, sets up an alert word to effectively mention the team.
     return !people.is_current_user(message.sender_email) && message.alerted;
 }
 
-export const initialize = (params) => {
+type AlertWords = {
+    alert_words: string[];
+};
```

Comment:
You may either make use of `interface` or `type` in the module until there is a specific need for one of those.